src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -4563,9 +4563,12 @@ protected StackValue POP_helper(Instruction instruction, out int blockId, out in
 
                 svData = rmem.Pop();
                 tempSvData = svData;
-                EX = PopToIndexedArray(blockId, (int)instruction.op1.opdata, (int)instruction.op2.opdata, dimList, svData);
+                PopToIndexedArray(blockId, (int)instruction.op1.opdata, (int)instruction.op2.opdata, dimList, svData);
             }
 
+#if TRACING_GC
```

Comment:
Is there a reason to run this off by default? Can we invert this so that everyone is getting the new GC?